server/utils/commonLocale.json

```diff
@@ -31,6 +31,7 @@
     "goalSummaryCard": {
       "info": {
         "aimToAchieveDate": "Aim to achieve this by {{ targetDate }}.",
+        "removedDate": "Removed on {{ achievedDate }}",
```

Comment:
Given we're now using it for the removed date we should probably rename "achievedDate" to "lastStatusChangeDate" both here and in goal-summary-card.njk